interop/alts/server/server.go

```diff
@@ -38,7 +43,12 @@ var (
 func main() {
 	flag.Parse()
 
-	lis, err := net.Listen("tcp", *serverAddr)
+	// If the server address starts with `unix:`, then we have a UDS address.
+	network := "tcp"
+	if strings.HasPrefix(*serverAddr, udsAddrPrefix) {
+		network = "unix"
+	}
+	lis, err := net.Listen(network, *serverAddr)
```

Comment:
I'm not sure, but do you need to also strip "unix:" from the address?
Does `net.Listen("unix", "unix:/tmp/blah")` work?

Reply:
@roseowl123

Reply:
net.Listen("unix", "unix:/tmp/blah") would work.

Reply:
Sorry, spoke too soon... I tried net.Listen("unix", "unix:/tmp/blah") in my test and it does not work.

Reply:
Thanks for checking this. Code is fixed.